forms/collections/elements.js

```diff
@@ -26,7 +27,11 @@ define(function (require) {
 
     if (elementModel.get('forms')) {
       errorList = elementModel.get('forms').reduce(function (errList, form) {
-        return form.get('elements').reduce(addToErrorList, errList);
+        elements = form.get('elements');
+        if (!elements) {
+          return [];
```

Comment:
Actually now that im looking at the code, Because the forms list is being reduced to an array, I think you need to return the accumulated `errList` here. I think returning `[]` would work for the first form, but wipe out anything previously accumulated if its not the first form.